deploy/docker-ephemeral/build/Makefile

```diff
@@ -0,0 +1,286 @@
+	sed -i.bak 's/tini\(.asc\|\)"/tini-\$$dpkgArch\1"/' $@
+
+airdock_rvm/Dockerfile:
+	git clone git@github.com:airdock-io/docker-rvm.git airdock_rvm-all
+	cd airdock_rvm-all && git reset --hard $(AIRDOCKRVM_COMMIT)
+	cp -R airdock_rvm-all/jessie-rvm airdock_rvm
+	sed -i.bak "s=airdock/base:jessie=$(USERNAME)/airdock_base$(TAGNAME)=" $@
+# add a second key used to sign ruby to the dockerfile. https://github.com/airdock-io/docker-rvm/issues/1
+	sed -i.bak "s=\(409B6B1796C275462A1703113804BB82D39DC0E3\)=\1 7D2BAF1CF37B13E2069D6956105BD0E739499BDB=" $@
+
+airdock_fakesqs/Dockerfile:
+	git clone git@github.com:airdock-io/docker-fake-sqs.git airdock_fakesqs-all
+	cd airdock_fakesqs-all && git reset --hard $(AIRDOCKFAKESQS_COMMIT)
+	cp -R airdock_fakesqs-all/0.3.1 airdock_fakesqs
+	sed -i.bak "s=airdock/rvm:latest=$(USERNAME)/airdock_rvm$(TAGNAME)=" $@
+# add a workdir declaration to the final switch to root.
+	sed -i.bak "s=^USER root=USER root\nWORKDIR /=" $@
+# break directory creation into two pieces, one run by root.
+	sed -i.bak "s=^USER ruby=USER root=" $@
+	sed -i.bak "s=cd /srv/ruby/fake-sqs.*=chown ruby.ruby /srv/ruby/fake-sqs\nUSER ruby\nWORKDIR /srv/ruby/fake-sqs\nRUN cd /srv/ruby/fake-sqs \&\& \\\\=" $@
+
+java_maven_node_python/Dockerfile:
+	git clone git@github.com:localstack/localstack.git java_maven_node_python
+	cd java_maven_node_python && git reset --hard $(JAVAMAVENNODEPYTHON_COMMIT)
+	cd java_maven_node_python && mv bin/Dockerfile.base Dockerfile
+# disable installing docker-ce. not available on many architectures in binary form.
+	sed -i.bak "/.*install Docker.*/{N;N;N;N;N;d}" $@
+
+localstack/Dockerfile:
+	git clone git@github.com:localstack/localstack.git localstack
+	cd localstack && git reset --hard $(LOCALSTACK_COMMIT)
+	sed -i.bak "s=localstack/java-maven-node-python=$(USERNAME)/java_maven_node_python$(TAGNAME)=" $@
+# skip tests. they take too long.
+	sed -i.bak "s=make lint.*=make lint=" localstack/Makefile
+	sed -i.bak "s=\(.*lambda.*\)=#\1=" localstack/Makefile
+
+minio/Dockerfile:
+	git clone git@github.com:minio/minio.git minio
+	cd minio && git reset --hard $(MINIO_COMMIT)
+
+cassandra/Dockerfile:
+	git clone git@github.com:docker-library/cassandra.git cassandra-all
+	cd cassandra-all && git reset --hard $(CASSANDRA_COMMIT)
+	cp -R cassandra-all/3.11 cassandra
+	 # work around go compiler bug by using newer version of GOSU. https://bugs.launchpad.net/qemu/+bug/1696353
+	sed -i.bak "s/GOSU_VERSION .*/GOSU_VERSION 1.11/" $@
+	 # add a block to the entrypoint script to interpret CS_JVM_OPTIONS, modifying the jvm.options before launching cassandra.
+	 # first, add a marker to be replaced before the last if.
+	sed -i.bak -r ':a;$$!{N;ba};s/^(.*)(\n?)fi/\2\1REPLACEME\nfi/' cassandra/docker-entrypoint.sh
+	 # next, load our variables.
+	sed -i.bak 's/REPLACEME/\nAPP_CONFIG="$$CASSANDRA_CONFIG"\n&/' cassandra/docker-entrypoint.sh
```

Comment:
It looks like this and the next lines all replace the same `REPLACEME` text; doesn't the first one replace all of them? Is there anything left for the following two commands?

Reply:
the '&' at the end of the line places the REPLACEME back in the text.